TombEngine/Game/control/control.h

```diff
@@ -1,4 +1,5 @@
 #pragma once
+#include <sdl3/SDL.h>
```

Comment:
Small first note, I encourage everyone to keep library includes like this in `framework.h` with the others. :D

Reply:
I think it's already included, so this one may be safely removed I guess?

Reply:
Done. I will remove others when found. SDL3 is included in framework.h